assets/src/containers/FilesAccessed.js

```diff
@@ -115,10 +119,24 @@ function FilesAccessed (props) {
     if (loaded) {
       if (filesDefaultData.default !== '') {
         setGradeRangeFilter(filesDefaultData.default)
+        let tempArray = []
+        resource_values.forEach(function(resource_item) {
+          if (resource_item.disabled === "false") {
+            tempArray.push(resource_item.resource_value)
+          }
+        })
+        setResourceFilter(resourceFilter.concat(tempArray))
         setDefaultValue(filesDefaultData.default)
       } else {
         // setting it to default
         setGradeRangeFilter('All')
+        let tempArray = []
+        resource_values.forEach(function(resource_item) {
+          if (resource_item.disabled === "false") {
+            tempArray.push(resource_item.resource_value)
+          }
+        })
+        setResourceFilter(resourceFilter.concat(tempArray))
```

Comment:
Code duplication should be avoided. Take the code as a separate function and reuse respective places

Reply:
Ok I will fix the code duplication. For when no resource type is selected, I thought it would make sense to update the chart to show no resources because no resource types are selected.

Reply:
yes, but if you know from UI they did not select any type then API call should not be allowed as you backend endpoint will fetch no data. We should preserve DB call as much as possible. I think when you see no resource type is selected, display a message saying "please select a resource type for data display"

Reply:
<img width="1423" alt="Screen Shot 2019-07-10 at 10 59 27 AM" src="https://user-images.githubusercontent.com/33735083/60980165-f60f3f80-a301-11e9-9c63-6ea337cd0572.png">

Should I center the message?

Reply:
No need to center but need to make it effective for the user to see. Right now it blends in with rest of the text and the info text might miss the attention. Try and see how can you make it effective for viewable